.circleci/release_s3.sh

```diff
@@ -8,7 +8,7 @@ echo "Releasing docs website"
 tar -xzf /tmp/workspace/deep-docs.tar.gz -C /tmp/workspace && aws s3 sync /tmp/workspace/docs s3://please-docs
 
 
-if aws s3 ls s3://please-releases/linux_amd64/$VERSION/; then
+if aws s3 ls s3://please-releases/linux_arm64/$VERSION/; then
```

Comment:
A little cheeky but this will have the current release sync the linux_arm64 files too